tokio/src/sync/mutex.rs

```diff
@@ -26,6 +26,11 @@
 //! }
 //! ```
 //!
+//! Note that in contrast to `std::sync::Mutex`, this implementation does not
+//! poison the mutex when a thread holding the `MutexGuard` panics. In such a
+//! case, the mutex will be unlocked. If the panic is caught, this might leave
+//! the data protected by the mutes in an inconsistent state.
```

Comment:
😅 `mutes` -> `mutex`?

Reply:
D'oh, sorry about that! I'll provide a fix.

Reply:
https://github.com/tokio-rs/tokio/pull/1934